package.json

```diff
@@ -8,9 +7,14 @@
   "es2015": "./esm2015/ngx-tabs-libdemo.js",
   "typings": "./ngx-tabs-libdemo.d.ts",
   "scripts": {
-    "build": "ng-packagr -p package.json",
+    "build": "bazel build //...",
     "copyPackageJson": "copyfiles package.json dist",
-    "semantic-release": "semantic-release pre && npm run copyPackageJson && npm publish dist && semantic-release post"
+    "semantic-release": "semantic-release pre && npm run copyPackageJson && npm publish dist && semantic-release post",
+    "postinstall": "ngc -p angular.tsconfig.json",
+    "preskylint": "bazel build --noshow_progress @io_bazel//src/tools/skylark/java/com/google/devtools/skylark/skylint:Skylint",
+    "skylint": "find . -type f \\( -name BUILD -or -name BUILD.bazel -or -name *.bzl \\) ! -path \"*/node_modules/*\" ! -path \"./dist/*\" | xargs $(bazel info bazel-bin)/external/io_bazel/src/tools/skylark/java/com/google/devtools/skylark/skylint/Skylint",
+    "prebuildifier": "bazel build --noshow_progress @com_github_bazelbuild_buildtools//buildifier",
+    "buildifier": "find . -type f \\( -name BUILD -or -name BUILD.bazel \\) ! -path \"*/node_modules/*\" | xargs $(bazel info bazel-bin)/external/com_github_bazelbuild_buildtools/buildifier/*/buildifier"
```

Comment:
I've read the description over here: https://github.com/alexeagle/angular-bazel-example/wiki/Formatting

So Buildifier does just formatting and checks (on CI if configured), a bit like the nrwl/nx setup does using prettier rules. Is that right?

Reply:
I think nx is switching to only run prettier on the regions of files you changed.
This does a global re-format and the CI does a global enforcement.